sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -36,6 +36,38 @@ enum class graph_state {
   executable,
 };
 
+// Forward declaration
+class node;
+
+namespace property {
+namespace graph {
+
+class no_cycle_check : public ::sycl::detail::DataLessProperty<
+                           ::sycl::detail::GraphNoCycleCheck> {
+public:
+  no_cycle_check() = default;
```

Comment:
Could we have a test that passes the `no_cycle_check` property to command_graph construction to check this is can be passed correctly? (test doesn't have to start trying to create cycles or anything)

Reply:
Makes sense. I've extended an existing test (empty node). Once implemented, we might want to extend the tests for cycle check.